bbz-project/src/components/ContextController.js

```diff
@@ -1,28 +1,39 @@
-import React, { createContext, useState } from "react";
+import React, { createContext, useState, useEffect } from "react";
 import useFetch from "../hooks/useFetch";
 
 const DataContext = createContext();
 
 const ContextController = (props) => {
     const [dbList, setDbList] = useState();
     const [dbNames, setDbNames] = useState();
-    const [currentDb, setCurrentDb] = useState({});
-    const { fetchDbNames, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
+    const [currentDbId, setCurrentDbId] = useState(0);
+    const [currentDb, setCurrentDb] = useState([]);
+    const { get, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
+
+    useEffect(async () => {
+        const response = await get(`/request_db?db=${currentDbId}`);
+        let newRecordsList = [];
+        for(let record in response.message) {
```

Comment:
Will be obsolete by Sunday, remember to delete the loop.

Reply:
of course sir, can i close the ticket yet?